src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -12,6 +21,30 @@
 	@PostConstruct
 	public void loadSomeData()
 	{
+		final File dir = new File("src/main/resources/venues/");
+		final File[] directoryListing = dir.listFiles();
+		final int count = 0;
+
+		if (directoryListing != null)
+		{
+
+			for (final File child : directoryListing)
+			{
+				try
+				{
+					final Scanner scanner = new Scanner(child);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Close this "Scanner". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2095)